authors/urls.py

```diff
@@ -13,12 +13,22 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import include, path
 from django.contrib import admin
+from django.urls import path, re_path, include
+
+from authors.apps.authentication.views import(
+    password_reset,
+    reset_password,
+    change_passowrd,
+)
 
 app_name = "authentication"
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include('authors.apps.authentication.urls')),
+    re_path(r'^api/set_password/complete/', change_passowrd, name='change_passowrd'),
+    re_path(r'^api/password_reset/', password_reset, name='password'),
```

Comment:
We upgraded this codebase from Django 1.1 to 2.1 , and one of the things we did change was to replace the `repath` to `path`, so i don't think it would be a good idea to return it again.